audb/core/load_to.py

```diff
@@ -228,6 +232,8 @@ def load_to(
         root: target directory
         name: name of database
         version: version string, latest if ``None``
+        cache_root: cache folder where databases are stored.
+            If not set :meth:`audb.default_cache_root` is used
```

Comment:
Here I also think we should be more specific on what is cached as the database will not be available inside the cache afterwards.

Reply:
maybe:

![image](https://user-images.githubusercontent.com/10383417/114551547-dbe75b80-9c63-11eb-9aec-23a1dabb1498.png)

Reply:
Yes, that would be sufficient.

Reply:
done